### ISSUE TITLE
Wrong tiles are loaded depending on map size

### ISSUE THREAD

**Issue by foobar3000** [wrobell/geotiler#23]
Creating rectangular map images (with fixed zoom level) results in messy/chaotic tile order on the map/image. 
The behavior doesn't always occur, possibly it depends on the coordinates.

Environment:
```
macOS
new python3.7 venv
geotiler 0.1.4 via pip
```

Simple test script:

```
# coding: utf-8
import sys
import geotiler
print ('geotiler ver.:', geotiler.__version__)
print ('python ver.:', sys.version)

pts = (8.44759, 49.0259355)  # somewhere in Germany.

for size in [(400, 400), (400, 300)]:
    mm = geotiler.Map(center=pts, size=size, zoom=13, provider="osm")
    image = geotiler.render_map(mm)
    image.save('map%s_%s.png' % size)
```

Console output:
```
geotiler ver.: 0.14.0
python ver.: 3.7.3 (default, Mar 27 2019, 09:23:15)
[Clang 10.0.1 (clang-1001.0.46.3)]
configuration file /Users/f3k/.config/geotiler/geotiler.ini does not exist
configuration file /Users/f3k/.config/geotiler/geotiler.ini does not exist
```

Wrong image (rectangular: 400x300px):
![map400_300](https://user-images.githubusercontent.com/398608/60126575-99067c00-978e-11e9-9b0d-db1adb8d5534.png)

Correct image (quadratic: 400x400px):
![map400_400](https://user-images.githubusercontent.com/398608/60126576-99067c00-978e-11e9-988f-0a1c9a9bedcd.png)

**Comment by wrobell** [comment]
Can also be replicated with
```
geotiler-lint -v -c 8.44759 49.0259355 -z 13 -s 400 300 test.png
```

**Comment by wrobell** [comment]
Fixed via a990e55e and version 0.14.1. Thanks for the report.
